deepcell_label_utils/utils.py

```diff
@@ -93,14 +99,100 @@ def int_coords(x):
     return img_mask
 
 
+class DCLZipConverter:
```

Comment:
This converter is likely to be integrated into cli/data_utils.py `load_data` function. The difference is that there would be no change of dimension and the initial parsing has to read byte from a .zip.dvc file. The name of the class might change to something more explicit like `DCLZipLoader `or `DCLZip2Dict` ?